### PR TITLE
WIP: Use page objects for user-organisations-list-test

### LINKED ISSUE #774: Rewrite integration tests for user-organizations-list component to use page objects
Falls under umbrella issue #697.

## Resources

- What's a page object? [[1](http://martinfowler.com/bliki/PageObject.html),[2](https://wyeworks.com/blog/2015/5/13/using-the-page-object-pattern-with-ember-cli/)]
- [Quickstart guide](http://ember-cli-page-object.js.org/docs/v1.6.x/quickstart)
- See [`tests/pages/components/donations/create-donation.js`](https://github.com/code-corps/code-corps-ember/blob/develop/tests/pages/components/donations/create-donation.js) and [`tests/integration/components/donations/create-donation-test.js`](https://github.com/code-corps/code-corps-ember/blob/develop/tests/pages/components/donations/create-donation.js) for in-house examples of, respectively, a page object and the page object being utilized in an integration test.
- To generate a page object, run `ember generate page-object component/<component-name>` in the project directory.

### PR DESCRIPTION
**Pull request by sarupbanskota**
Closes #774 
Also under #697

### CODE REVIEW

File: tests/integration/components/user-organizations-list-test.js [hunk 1]
```diff
@@ -1,15 +1,19 @@
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
+import page from '../../pages/components/user-skills-input';
```
Comment (sarupbanskota) [inline tests/integration/components/user-organizations-list-test.js]: what's wrong with importing `page` this way?

File: tests/integration/components/user-organizations-list-test.js [hunk 2]
```diff
@@ -49,12 +53,12 @@ test('with several organizations renders all required elements', function(assert
...
-  assert.equal(this.$('h2').text(), 'Organizations', 'The header renders');
-  assert.equal(this.$('li').length, 3, 'All organization items render');
-  assert.equal(this.$('li:first h3').text().trim(), 'Organization 1', 'The organization header renders');
-  assert.equal(this.$('li:first p').text().trim(), 'Organization 1 description', 'The organization description renders');
+  assert.ok(page.userOrganisationListVisible);
+  assert.equal(page.header, 'Organizations', 'The header renders');
+  assert.equal(page.listItemCount, 3, 'All organization items render');
+  assert.equal(page.header, 'Organization 1', 'The organization header renders');
+  assert.equal(page.description, 'Organization 1 description', 'The organization description renders');
   assert.equal(this.$('li:first img').attr('src'), '/icon_1.png', 'The organization image renders');
```
Comment (sarupbanskota) [inline tests/integration/components/user-organizations-list-test.js]: any tips on what prop I could use the get the `attr('src')` for a DOM element?

Comment (sarupbanskota) [inline tests/integration/components/user-organizations-list-test.js]: I see http://ember-cli-page-object.js.org/docs/v1.6.x/api/properties#property, just wanted to confirm if that's the same as `attr`?

**Comment by JoshSmith** [comment]
@sarupbanskota tests are failing here. Not sure why. Are they passing locally?

**Comment by begedin** [comment]
Hm, this one is failing again. Was passing before, I think, no?

**Pull request merged**